share/completions/sv.fish

```diff
@@ -0,0 +1,80 @@
+
+complete -kfc sv -n "$no_comm" -a check           -d "Check if the service is in it's requested state"
+complete -kfc sv -n "$no_comm" -a try-restart     -d "Run term, cont, and  up, report status"
+complete -kfc sv -n "$no_comm" -a force-shutdown  -d "Run exit, report status or kill on timeout"
+complete -kfc sv -n "$no_comm" -a force-restart   -d "Run term, cont and up, report status"
+complete -kfc sv -n "$no_comm" -a force-reload    -d "Run term and  cont, report status"
+complete -kfc sv -n "$no_comm" -a force-stop      -d "Run down, report status or kill on timeout"
+complete -kfc sv -n "$no_comm" -a shutdown        -d "Run exit, report status"
+complete -kfc sv -n "$no_comm" -a restart         -d "Run term, cont, and  up, report status using ./check"
+complete -kfc sv -n "$no_comm" -a reload          -d "Run hup, report status"
+complete -kfc sv -n "$no_comm" -a stop            -d "Run down, report status"
+complete -kfc sv -n "$no_comm" -a start           -d "Run up, report status"
+
+complete -kfc sv -n "$no_comm" -a e         -d "Alias for exit"
+complete -kfc sv -n "$no_comm" -a k         -d "Alias for kill"
+complete -kfc sv -n "$no_comm" -a t         -d "Alias for term"
+complete -kfc sv -n "$no_comm" -a 2         -d "Alias for 2"
+complete -kfc sv -n "$no_comm" -a 1         -d "Alias for 1"
+complete -kfc sv -n "$no_comm" -a q         -d "Alias for quit"
+complete -kfc sv -n "$no_comm" -a i         -d "Alias for interrupt"
+complete -kfc sv -n "$no_comm" -a a         -d "Alias for alarm"
+complete -kfc sv -n "$no_comm" -a h         -d "Alias for hup"
+complete -kfc sv -n "$no_comm" -a c         -d "Alias for cont"
+complete -kfc sv -n "$no_comm" -a p         -d "Alias for pause"
+complete -kfc sv -n "$no_comm" -a exit      -d "Send TERM, and CONT, report status"
+complete -kfc sv -n "$no_comm" -a kill      -d "Send SIGKILL"
+complete -kfc sv -n "$no_comm" -a term      -d "Send SIGTERM"
+complete -kfc sv -n "$no_comm" -a 2         -d "Send SIGUSR2"
+complete -kfc sv -n "$no_comm" -a 1         -d "Send SIGUSR1"
+complete -kfc sv -n "$no_comm" -a quit      -d "Send SIGQUIT"
+complete -kfc sv -n "$no_comm" -a interrupt -d "Send SIGINT"
+complete -kfc sv -n "$no_comm" -a alarm     -d "Send SIGALRM"
+complete -kfc sv -n "$no_comm" -a hup       -d "Send SIGHUP"
+complete -kfc sv -n "$no_comm" -a cont      -d "Send SIGCONT"
+complete -kfc sv -n "$no_comm" -a pause     -d "Send SIGSTOP"
+
+complete -kfc sv -n "$no_comm" -a o      -d "Alias for once"
+complete -kfc sv -n "$no_comm" -a d      -d "Alias for down"
+complete -kfc sv -n "$no_comm" -a u      -d "Alias for up"
+complete -kfc sv -n "$no_comm" -a s      -d "Alias for status"
+complete -kfc sv -n "$no_comm" -a once   -d "Start service, but don't restart it"
+complete -kfc sv -n "$no_comm" -a down   -d "Send it the TERM signal"
+complete -kfc sv -n "$no_comm" -a up     -d "Start a service"
+complete -kfc sv -n "$no_comm" -a status -d "Report the current status of the service"
```

Comment:
huh, so with the `-k` option, the arguments are listed in reverse order. Interesting!
(it's usually used for preserving order within a single `complete` entry)

Reply:
I just wish it was not reversed :confused:

Reply:
.yeah it seems a bit strange. This is mostly used in Git completions AFAIK